src/footprints/via.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+    nets: ['from'],
+    body: p => `
+
+	(module VIA-0.6mm (layer F.Cu) (tedit 591DBFB0)
+	
+        ${p.at /* parametric position */}   
+		  (fp_text reference REF** (at 0 1.4) (layer F.SilkS) hide
+			(effects (font (size 1 1) (thickness 0.15)))
+		  )
+		  (fp_text value VIA-0.6mm (at 0 -1.4) (layer F.Fab) hide
+			(effects (font (size 1 1) (thickness 0.15)))
+		  )
+		  (pad 1 thru_hole circle (at 0 0) (size 0.6 0.6) (drill 0.3) (layers *.Cu) (zone_connect 2) ${p.net.from})

```

Comment:
What the hell is a `zone_connect`? Never seen one of these before... :open_mouth: